services/src/main/java/org/exoplatform/addons/gamification/service/configuration/BadgeService.java

```diff
@@ -47,6 +47,28 @@ public BadgeDTO findBadgeByTitle(String badgeTitle) {
         }
         return null;
 
+    }
+    /**
+     * Find a BadgeEntity by id
+     * @param badgeTitle : badge id
```

Comment:
This is causing build failure for the PR
```suggestion
     * @param badgeId : badge id
```